### DISCUSSION TITLE
v3 Test Download

### DISCUSSION THREAD

**Discussion by IceHacks** [IceHacks/SurvivCheatInjector#746] [Announcements]
This is not an official release, but rather an easier-to-access test. The initial testing on Telegram has been fun and I am almost ready to make official releases. The official releases will start with version `3.1.0`. The cheat auto-updates, so this version will be the only test version I will publish here. Hopefully, we will add more features and improvements. Let me know if you want to assist in the development.

If you do not know much about the 4th cheat rewrite, here is the general breakdown:

**It aims to improve surviv performance**
Kongregate has not been paying much attention to this issue, so I have.

**It aims to provide a good aimbot for all weapons**
If autoaim is enabled, it accomplishes this. Work still must be done for aim-on-click and the new aimbot.

**It aims to clean up the surviv UI**
Surviv was once a simple game, they really messed up the starting page. This program simplifies the menu.

**It aims to be modular**
The cheat will have a plugin loading UI, along with cool built-in features (like a bot sender).

Of course, not everything that I want to happen is listed here and not all of it will be added before 3.1, but I hope that over time we can make a cheat that gets Kongregate's attention. I want them to be forced to make their game better. That is why I am back.

_Download the file appropriate for your system (EXE for Windows, AppImage for Linux, DMG for MacOS)._
###### &copy; 2021 IceHacks

<hr /><em>This discussion was created from the release <a href='https://github.com/IceHacks/SurvivCheatInjector/releases/tag/v3.0.6'>v3 Test Download</a>.</em>

**Comment by criticalguy24234** [comment]
i have problems with this test version first is flashlight not works i mean after playing for like 1 or 2 games it will broke pls fix
also add auto attack on control so i can press LMB to attack

**Reply by ghost** [reply]
oof

**Reply by criticalguy24234** [reply]
im using windows

**Reply by IceHacks** [reply]
Thanks for the bug report and idea! That bug is slated to be fixed, and that idea is of interest. I believe the current mechanics behind the auto attack is that it attacks whenever aim is active. I will add an option to change that.

**Reply by phdflopper07** [reply]
wait so how do you install it

**Reply by sssssse** [reply]
> ¡Gracias por el informe de errores y la idea! Ese error está programado para ser corregido, y esa idea es de interés. Creo que la mecánica actual detrás del ataque automático es que ataca siempre que el objetivo está activo. Agregaré una opción para cambiar eso.

ayudameeeeeeeeeeeee no se abre el injector

**Reply by sofianepro12** [reply]
idk if i can get virus

**Comment by ghost** [comment]
What's the bot's panel? 
Linux version.

**Reply by flancast90** [reply]
@lillabell, it works fine for me on Linux using the .appimage file. (right click after download > permissions > allow as executable file)

**Reply by IceHacks** [reply]
I am working on implementing a bots panel. It is supposed to hide itself on boot, maybe it does not do that on Linux.

**Reply by ghost** [reply]
@IceHacks cool but what does it do?

**Comment by flancast90** [comment]
Nice job @IceHacks! Looks great and works wonderfully!

**Reply by IceHacks** [reply]
Thanks!

**Reply by wdgohdw** [reply]
i can use it bcs can not send to extension,how to make it work?

**Reply by sssssse** [reply]
hola una pregunta el icehacks tiene virus noo carga la extension ¿soluciones?

**Reply by Mrch3zz** [reply]
@IceHacks which do i download i am on chromebook

**Comment by iBLiSSIN** [comment]
did you add autoaim and fix auto shoot?

**Comment by Sergio134** [comment]
Just wondering, will the full release be an extension or a computer download like how it is now. for the release, I would like to toggle the auto break because when I am fighting my character switches to fist and I get killed. also, will you see the simple menu in the full release? because being honest, I'm already used to the new UI. Thank you!

**Reply by IceHacks** [reply]
I am planning on keeping the current UI implemented in v3. It will be an executable, as that is most convenient for me. However, we are looking into other formats.

**Reply by dudethis** [reply]
> I am planning on keeping the current UI implemented in v3. It will be an executable, as that is most convenient for me. However, we are looking into other formats.

good work on the new updates, hope you make another format :)

**Comment by TylerWhat** [comment]
this version is great!

**Comment by Evan5731** [comment]
I can't put it in  chrome://extensions/ D:

**Reply by iBLiSSIN** [reply]
> I can't put it in chrome://extensions/ D:

It is a application, not extensions

**Reply by anthonyrosero2005** [reply]
Me neither, but if you look at videos it comes out that it has to be put in chrome extensions

**Reply by anthonyrosero2005** [reply]
is for windows

**Comment by Person441** [comment]
can u remove the annoying popups
![image](https://user-images.githubusercontent.com/91803184/135753020-356bb458-b399-40b3-8b0e-4ab3c6b4acd5.png)

**Reply by Onacona-BG** [reply]
Mac OS, why does the app keep loading when i open it, and it doesn't stop, its been 30 minutes now :(

**Reply by IceHacks** [reply]
> Mac OS, why does the app keep loading when i open it, and it doesn't stop, its been 30 minutes now :(

Hey there! I don't own a Mac and cannot test it, but I will try and get an answer for you.

**Reply by Onacona-BG** [reply]
> > Mac OS, why does the app keep loading when i open it, and it doesn't stop, its been 30 minutes now :(
> 
> Hey there! I don't own a Mac and cannot test it, but I will try and get an answer for you.

your so nice, thx alot XD

**Reply by TylerWhat** [reply]
maybe u can check your internet😊

**Reply by Onacona-BG** [reply]
> maybe u can check your internet😊

i did, genius

**Reply by IceHacks** [reply]
Possibly try rebooting the app multiple times until it starts. It should start within 5 minutes.

**Comment by wdgohdw** [comment]
how to download.__. i can't put it in extension:V

**Reply by wdgohdw** [reply]
your cheat is freaking good,btw

**Reply by IceHacks** [reply]
Did you figure it out? What OS do you have?

**Reply by wdgohdw** [reply]
window os:V

**Reply by wdgohdw** [reply]
i'm still stuck at 2.1.7

**Reply by TylerWhat** [reply]
> i'm still stuck at 2.1.7
you should download the bag it's an app

**Comment by randomguylmao** [comment]
Which one is for a chromebook?

**Reply by Onacona-BG** [reply]
no chromebook
:(

**Reply by IceHacks** [reply]
Hey there! ChromeOS has some compatibility issues when it comes to developing apps, but it is possible that you could install the Linux version on ChromeOS.

**Reply by randomguylmao** [reply]
I Installed the linux version, but whenever I play, its very laggy. Is there anything I can do to reduce/get rid of the lag?

**Reply by IceHacks** [reply]
> I Installed the linux version, but whenever I play, its very laggy. Is there anything I can do to reduce/get rid of the lag?

Try enabling some performance options.

**Comment by anthonyrosero2005** [comment]
help, I installed the file I tried to add it in extension and it does not come out, then I unzipped it and tried to add it and I get this:
The manifest file is missing or cannot be read
The manifest file could not be loaded.

**Reply by anthonyrosero2005** [reply]
@IceHacks help me please :(

**Reply by ghost** [reply]
@anthonyrosero2005 what do you mean by "extension"? Isn't this about the application?

**Reply by TylerWhat** [reply]
no you shouldn't put the file on chrome extension, the link is for download an app

**Comment by Onacona-BG** [comment]
@IceHacks Is it possible to provide a new discord server link, none of the older ones are working :(

**Reply by anthonyrosero2005** [reply]
help me install the hack please :(

**Comment by xsearhaj** [comment]
@IceHacks Will a version on chrome come anytime soon?

**Reply by IceHacks** [reply]
Doubt it, sorry

**Comment by Sergio134** [comment]
Also, I forgot to mencion will you make this app for mobile because mobile players have a lot of performance issues. Thank you!

**Reply by IceHacks** [reply]
I was thinking of it 😉

**Reply by Sergio134** [reply]
epic. this reply makes me happy!

**Comment by lil-tecca** [comment]
@IceHacks Could you add a speed hack to make you move and punch faster maybe even use healing items/reload quicker than the wait time?

**Reply by IceHacks** [reply]
The timers are all on the server side, so, unless surviv has a bug I don't know of, that is impossible.

**Reply by Sergio134** [reply]
I don't think this is a good idea because then people will know that you are hacking when in first impressions, this could lead to banning or account termination

**Comment by Fiery5029** [comment]
Hey, great job on the release but how would I go about downloading this on Chrome OS?

**Reply by humphreygaming** [reply]
Try downloading the file for Linux

**Reply by BlueMinty** [reply]
Which file is it for chrome

**Comment by humphreygaming** [comment]
@BlueMinty none, no downloads for that

**Reply by ghost** [reply]
thats not how you're supposed reply. you didn't do it properly >:(

**Comment by sssssse** [comment]
no abre :(

**Comment by IExpIoit** [comment]
I can't download it, i use window and it say can't open the exe pls help

**Comment by TechDude100** [comment]
I did a scan on virus total and It says Suspicious.Win32.Save.a and W32.Hack.Tool. Is this correct or is it safe? Thanks!

**Reply by humphreygaming** [reply]
False positive, Microsoft really doesn't like hacking tools

**Reply by lionstone1** [reply]
@humphreygaming But what's the matter with the W32.Hack.Tool ? virustotal.com also detects the same.

**Comment by Angel-IA** [comment]
I'm using a chromebook, Pleas help

**Reply by ghost** [reply]
bruh it's for mac, windows, or linux only

**Reply by penguin20154** [reply]
if you are using a Chromebook use 2.17 instead.

**Comment by NotWorkingAlways** [comment]
2."Click on "latest.zip" under "Assets", you will download the cheats ZIP". i cant find latest.zip under assest

**Comment by ghost** [comment]
OK so how tf do I do this? because it's just taking me to an archive and the latest file, when launched says that there is a new update, and I'm taken back to the first page. I really need the file, I cannot find it

**Comment by penguin20154** [comment]
can you make it available to chrome? cuz I'm stuck with using 2.17 for now.

**Comment by Gide0nXD22** [comment]
How do I use it on a school iPad it’s an apple brand iPad

**Reply by IceHacks** [reply]
I don't believe you can, in any way or version.

**Comment by Gide0nXD22** [comment]
Ok thx

Sent from my iPad

On Nov 30, 2021, at 7:14 AM, IceHacks ***@***.***> wrote:

﻿

I don't believe you can, in any way or version.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/IceHacks/SurvivCheatInjector/discussions/746#discussioncomment-1722153>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AWWIQXSCTNRYDEBV2SPSZNLUOTE4DANCNFSM5FGUNA7A>
.
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
or Android
<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by KenshiroYTB** [comment]
This is cool, but can you combine this with the Surviv.io Custom Sound? You are one of the creator of this so i guess the next update the custom sound will be added to the cheat

**Reply by humphreygaming** [reply]
I'd rather prefer that SCS-RP (and related) be added as an optional module that could be installed via a link or local file. Adding custom sounds to the whole userbase could prove problematic when one disagrees on the audio / stylistic choices.

**Comment by Dangello200** [comment]
which of all the files is the one that I have to download, please help @IceHacks

**Reply by IceHacks** [reply]
Download the file appropriate for your system (EXE for Windows, AppImage for Linux, DMG for MacOS).

**Comment by zane12345** [comment]
What if I'm running this on a Chrome OS device?

**Reply by IceHacks** [reply]
You won't be. This app can only run on supported platforms. Unless you have administrator control of your Chrome OS, you cannot use it.
